LocalMultiplayerAgent/Globals.cs

```diff
@@ -30,6 +30,11 @@ public static class Globals
 
         public static GameServerEnvironment GameServerEnvironment { get; set; }
 
+        public const string ZipExtension = ".zip";
```

Comment:
We should move this to the Core library. So that the Asset extractor has access to these directly, without needing to be passed in (otherwise we will have to pass in again from actual VnAgent as well). Especially these are static strings that don't change so all classes should have access to them.

Reply:
Thanks. But there are some other parts still need global file extension variables. For example in GameResourceManger.cs, there is SupportedAssetArchiveFormats variable ValidateAssetType() method still need to access it (VmAgent). what about creating a static class has these 3 file extension variables? Or should we keep that in VmAgent Side?

Reply:
We should move it to the Core library. We can have a static class called Constants.cs and put those in there.